src/config.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AuthenticationConfig, Common, ServiceConfig} from '../third_party/types/common-types';
+import {ProfilerConfig} from './profiler';
+
+const common: Common = require('@google-cloud/common');
+const extend = require('extend');
+
+// Configuration for Profiler.
+export interface Config extends AuthenticationConfig {
+  // Cloud Console projectId to associate profiles with instead of one read
+  // from VM metadata server.
+  projectId?: string;
+
+  // Log levels: 0-disabled,1-error,2-warn,3-info,4-debug.
+  // Defaults to value of 1.
```

Comment:
I'd drop the default values from the doc now and just mention in the documentation to the Config type that the default values can be found below in defaultConfig. I know we are going back and forth on this, this is hopefully last time :)